contracts/schemes/Locking4Reputation.sol

```diff
@@ -43,9 +43,9 @@ contract Locking4Reputation {
     /**
      * @dev redeem reputation function
      * @param _beneficiary the beneficiary for the release
-     * @return bool
+     * @return uint reputation rewarded
      */
-    function redeem(address _beneficiary) public returns(bool) {
+    function redeem(address _beneficiary) public returns(uint) {
```

Comment:
function redeem(address _beneficiary) public returns(uint reputation)

and remove reputation definition inside the function.

Reply:
done: https://github.com/daostack/arc/pull/566/commits/2418b4276e2160c1936b493be377d5631215430c